src/locales/locale-en.json

```diff
@@ -401,16 +410,16 @@
     "TITLE": "Map"
   },
   "HISTORY": {
-    "ACTIVITY_CREATE": "created a new one-time pick-up",
-    "ACTIVITY_DELETE": "deleted a pick-up",
-    "ACTIVITY_DISABLE": "disabled a pickup",
-    "ACTIVITY_DONE": "did a pick-up at {storeName}",
-    "ACTIVITY_ENABLE": "enabled a pickup",
-    "ACTIVITY_JOIN": "signed up to do a pick-up at {storeName}",
-    "ACTIVITY_LEAVE": "stepped back from doing a pick-up at {storeName}",
+    "ACTIVITY_CREATE": "created a new one-time activity",
+    "ACTIVITY_DELETE": "deleted an activity",
+    "ACTIVITY_DISABLE": "disabled an activity",
+    "ACTIVITY_DONE": "did an activity at {storeName}",
```

Comment:
Change `{storeName}` to `{placeName}`

Reply:
Done... given we're changing the message anyway, might as well change this. Although pending any ideas about how to minimise translation burden for this change...